.github/workflows/javascript.yml

```diff
@@ -17,20 +10,26 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-        name: Checkout files
+      - name: Checkout files
+        uses: actions/checkout@v3
+        with:
+          sparse-checkout: |
+            .github
+            javascriptv3
       - name: Get changed files
         id: changed-files
         uses: tj-actions/changed-files@v41
         with:
-          files: "javascriptv3/**/*.{js,ts,jsx,tsx}"
+          files: "javascriptv3/**/*.js"
```

Comment:
We should be linting ts,jsx,and tsx as well. Those are also "javascript" files.

Reply:
I mean. I'm not even sure why I changed that :P